dnf/cli/output.py

```diff
@@ -1474,12 +1474,12 @@ def historyListCmd(self, extcmds):
                 elif old.return_code:
                     rmark = lmark = '#'
                     # We don't check .errors, because return_code will be non-0
-                elif old.output:
+                elif old.is_output:
```

Comment:
Transaction output is not loaded by default. `is_output` is boolean that indicates whether there is an output available for that transaction.